custom_metrics/metric-summary.md

```diff
@@ -0,0 +1,706 @@
+# Summary of all custom metrics
+
+Here you will find a summary of all the metrics used in their respective custom metrics file
+
+# [almanac.js](https://github.com/HTTPArchive/legacy.httparchive.org/blob/master/custom_metrics/almanac.js) metrics
+
+## 01.12
+Whether the page contains any instances of <script type=module>.
+
+`1` if true, `0` if false
+
+## 01.13
+Whether the page contains any instances of  <script nomodule>
+
+`1` if true, `0` if false
+
+## link-nodes
+A JSON array of <link> nodes used in the <head> element.
+
+Example response:
+
+```json
+{
+  "total": 28,
+  "nodes": [
+    {
+      "tagName": "link",
+      "rel": "stylesheet",
+      "href": "/static/css/normalize.css?v=112272e51c80ffe5bd01becd2ce7d656"
+    },
+    {
+      "tagName": "link",
+      "rel": "preload",
+      "href": "/static/css/normalize.css?v=112272e51c80ffe5bd01becd2ce7d656",
+      "as": "style"
+    },
+    ...
+  ],
+  "attribute_usage_count": {
+    "rel": 28,
+    "href": 28,
+    "as": 8,
+    "type": 5,
+    "crossorigin": 5,
+    "hreflang": 14
+  }
+}
+```
+
+## priority-hints
+A JSON array of prioritized nodes and their key/value attributes. Checks for the `importance` attribute in `<link>` `<img>` `<script>` or `<iframe>`.
+
+Example response:
+
+```json
+{
+  "total": 28,
+  "nodes": [
+    {
+      "tagName": "link",
+      "rel": "...",
+      "href": "...",
+      "importance": "low",
+    },
+    {
+      "tagName": "img",
+      "href": "...",
+      "importance": "low",
+    },
+    ...
+  ],
+  "attribute_usage_count": {
+    "rel": 28,
+    "href": 28,
+    "as": 8,
+    "type": 5,
+    "importance": 32
+  }
+}
+```
+
+## meta-nodes
+A JSON array of `<meta>` nodes within the `<head>` element.
+
+Example response:
+
+```json
+"meta-nodes": {
+  "total": 16,
+  "nodes": [
+    {
+      "tagName": "meta",
+      "charset": "UTF-8"
+    },
+    {
+      "tagName": "meta",
+      "name": "viewport",
+      "content": "width=device-width, initial-scale=1"
+    },
+    {
+      "tagName": "meta",
+      "name": "description",
+      "content": "The Web Almanac is an annual state of the web report combining the expertise of the web community with the data and trends of the HTTP Archive."
+    },
+    ...
+  ],
+  "attribute_usage_count": {
+    "charset": 1,
+    "name": 8,
+    "content": 15,
+    "property": 7
+  }
+}
+```
+
+## 10.5
+
+*DEPRECATED*
```

Comment:
How should we think about deprecated metrics? Are there always better replacements somewhere else (if so where) or should we delete them?

Reply:
Yes, both of those deprecated metrics are in the SEO chapters metrics. Once the wpt_bodies.js file is written in here, we can link that the new version can be found there.

Reply:
Ok SGTM. Could you add a link to that JS file so if someone is looking to query this data and finds this documentation first, they know where to look instead? Once we have docs for the other file it'll be easier to link together but we can link to the source JS itself in the mean time.

Reply:
Done